cmd/bbb_frontend/main.go

```diff
@@ -36,14 +36,15 @@ func main() {
 	var (
 		actionCacheAllowUpdates = flag.Bool("ac-allow-updates", false, "Allow clients to write into the action cache")
 		blobstoreConfig         = flag.String("blobstore-config", "/config/blobstore.conf", "Configuration for blob storage")
+		metricsPort             = flag.String("metrics", ":80", "port to serve metrics on")
```

Comment:
```suggestion
		metricsPort             = flag.String("metrics-port", ":80", "Port on which to serve metrics")
```

Same for the other ports.